### PR TITLE
Added APIView json token gist file

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Moved apiview token gist file to tools repo and added new property field for cross language ID

### CODE REVIEW

File: src/dotnet/APIView/apiview_token_gist.json [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1",
+            "NavigateToId": null,
+            "Value": "Class1",
+            "Kind": 6,
+            "CrossLanguageDefId": "Namespace1.Class1"
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/apiview_token_gist.json:110]: @lmazuel @weshaggard Here is the new token property added for common ID as per our earlier discussion.

File: src/dotnet/APIView/apiview_token_gist.json [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1.Class1(System.String, System.Int32)",
+            "NavigateToId": null,
+            "Value": "Class1",
+            "Kind": 6,
+            "CrossLanguageDefId": "Namespace1.Class1.Class1"
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/apiview_token_gist.json:159]: @lmazuel and this is for method/constructor level. This ID format can be used for public properties/variables  within a class also.

**Review by lmazuel — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

File: src/dotnet/APIView/apiview_token_gist.json (hunk 2 above)
Comment (tjprescott) [reply src/dotnet/APIView/apiview_token_gist.json:159]: If there is an overload of this method, how would the CrossLangaugeDefId be a) deterministic b) unique?

Comment (praveenkuttappan) [reply src/dotnet/APIView/apiview_token_gist.json:159]: Then we need further classifier with a number suffix if every language will have taht overloaded API.

**Pull request merged**